WordPress/Classes/ViewRelated/Me/Me Main/MeViewController.swift

```diff
@@ -12,7 +12,6 @@ class MeViewController: UITableViewController {
 
     override init(style: UITableView.Style) {
         super.init(style: style)
-        navigationItem.title = NSLocalizedString("Me", comment: "Me page title")
```

Comment:
nitpick

Should we keep "Me" in the navigation bar given the navigation bar appears when scrolling up?

Apple Id settings keep the title:

https://github.com/wordpress-mobile/WordPress-iOS/assets/4062343/3f964692-a1b6-4c84-af16-8ab62d43500f

While GitHub app makes the title appear only when scrolling up:

https://github.com/wordpress-mobile/WordPress-iOS/assets/4062343/1b11e595-e3d2-4d3e-ac1d-75385f4bb1b1

Reply:
Sure, I'll add it back for now. It could be updated if a more comprehensive redesign comes.